src/lib/api/utils.js

```diff
@@ -146,3 +146,12 @@ export const searchReady = async () => {
     setTimeout(resolve, invenioConfig.APP.SEARCH_READY_DELAY)
   );
 };
+
+/**
+ * Get the total number of results from a search API call, to
+ * support ES 6 and ES 7.
+ * @param {*} responseData the response data object
+ * @returns the total number of results
+ */
+export const getSearchTotal = (hits) =>
```

Comment:
in the long term as a solution, shouldn't this be the responsibility of the backend to normalise it? f.e. invenio-search, to have the consistent structure on REST, not depending on the ES version? Should frontend be avare of ES version?

Reply:
Very valid point! You are totally right!